### PR TITLE
Avoid self-XSS

### PR DESCRIPTION
**Pull request by bfred-it**
Classic `innerHTML = userString`

### CODE REVIEW

File: extension/api.js [hunk 1]
```diff
@@ -36,3 +36,20 @@ HideFilesOnGitHub.storage = {
 		chrome.storage.sync.set(object);
 	}
 };
+
+// Inlined partial `escape-goat` package
+window.escape = input => input
```
Comment (sindresorhus) [inline extension/api.js]: You should probably not override the `escape` global.
```suggestion
window.escapeHtml = input => input
```

File: extension/api.js [hunk 2]
```diff
@@ -36,3 +36,20 @@ HideFilesOnGitHub.storage = {
...
+	.replace(/&/g, '&amp;')
+	.replace(/"/g, '&quot;')
+	.replace(/'/g, '&#39;')
+	.replace(/</g, '&lt;')
+	.replace(/>/g, '&gt;');
+
+window.escapeTag = function (input, ...parts) {
```
Comment (sindresorhus) [inline extension/api.js]:
```suggestion
window.escapeTag = (input, ...parts) => {
```

File: extension/api.js (hunk 1 above)
Comment (sindresorhus) [reply extension/api.js]: Doesn't really need to be on `window` at all as it's only used in this file.

File: extension/api.js [hunk 3]
```diff
@@ -38,17 +38,17 @@ HideFilesOnGitHub.storage = {
...
-window.escape = input => input
+const escapeHTML = input => input
 	.replace(/&/g, '&amp;')
 	.replace(/"/g, '&quot;')
 	.replace(/'/g, '&#39;')
 	.replace(/</g, '&lt;')
 	.replace(/>/g, '&gt;');
 
-window.escapeTag = (input, ...parts) => {
+window.escapeTag = function (input, ...parts) {
```
Comment (sindresorhus) [inline extension/api.js]: Huh? This got reverted?

Comment (bfred-it) [reply extension/api.js]: Ah yeah, I opened an _edit_ page before merging your other suggestion.

**Pull request merged**